Clarifications in Okta SSO doc (#21777)
gravitational/teleport: docs/pages/access-controls/sso/okta.mdx

```
The above example includes the `entity_descriptor` field as multiple lines, although a single line with no spaces will also be accepted.

> The `attributes_to_roles` listing in the Spec maps key/value-like attributes of

Reviewer: ```suggestion
The `attributes_to_roles` field in the connector resource maps key/value-like attributes of
```

> the assertion from Okta into a list of Teleport Roles to apply to the session.

Reviewer: ```suggestion
the assertion from Okta into a list of Teleport roles to apply to the session.
```

We are going to create a new role that'll pull in external information from Okta. Notice
`{{external.username}}` login. It configures Teleport to look at *"username"* Okta claim
and use that field as an allowed login for each user.  This example uses email as the
username format.  The `email.local(external.username)` function call will remove

> the `@domain` and just have the username prefix.

Reviewer 2: ```suggestion
the `@domain` and leave the username prefix.
```
